DDR4 OC Guide.md

```diff
@@ -271,7 +271,7 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
   | B | Hynix 8Gb CJR, Samsung 4Gb E-Die, Nanya 8Gb B-Die | High end ICs with the ability to run high frequencies with good timings. |
   | C | Hynix 8Gb JJR, Hynix 16Gb MJR, Hynix 16Gb CJR, Micron 16Gb Rev. E, Samsung 8Gb D-Die, Nanya 8Gb A-Die | Decent ICs with good performance and decent frequency scaling.|
   | D | Hynix 8Gb AFR, Hynix 8Gb BFR, Micron 8Gb Rev. B, Micron 8Gb Rev. H, Samsung 8Gb C-Die, Samsung 4Gb D-Die | Low end ICs commonly found in average cheap kits. Most are EOL and no longer relevant. 
```

Comment:
```suggestion
  | D | Hynix 8Gb AFR, Micron 8Gb Rev. B, Samsung 8Gb C-Die, Samsung 4Gb D-Die | Low end ICs commonly found in average cheap kits. Most are EOL and no longer relevant. 
```

Can't seem to find much info on these ICs so it's best not to rank them until more about them is known

Reply:
There's sources for Rev H in the OC discord ram OC chat. It's either profile name, dabo or similar who had some. Peanut I think commented on it too.